pkg/postgres/schema/log_imbues.go

```diff
@@ -0,0 +1,48 @@
+// Code generated by pg-bindings generator. DO NOT EDIT.
+
+package schema
+
+import (
+	"reflect"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/walker"
+)
+
+var (
+	// CreateTableLogImbuesStmt holds the create statement for table `log_imbues`.
+	CreateTableLogImbuesStmt = &postgres.CreateStmts{
+		Table: `
+               create table if not exists log_imbues (
+                   Id varchar,
+                   serialized bytea,
+                   PRIMARY KEY(Id)
+               )
```

Comment:
Why is the timestamp field missing here? 🤔

Reply:
I didn't mark it with search currently. Will only do that once we actually use those endpoints

Reply:
Ah, of course.